docs/aws/audit/cloudwatchmonitoring/rules/cloudwatch_log_groups_not_encrypted.mdx

```diff
@@ -22,98 +22,72 @@ HIPAA,PCIDSS,GDPR,CISAWS,CBP,NIST,SOC2,AWSWAF,SEBI,RBI_UCB
-
-#### Using CLI
-
-To remediate the misconfiguration of CloudWatch Log Groups not being encrypted in AWS using AWS CLI, follow these steps:
-
-1. List all the CloudWatch Log Groups that are not encrypted:
-```bash
-aws logs describe-log-groups --query 'logGroups[?kmsKeyId==null].logGroupName' --output text
-```
-
-2. For each unencrypted CloudWatch Log Group, update the log group to enable encryption using a KMS key:
-```bash
-aws logs put-retention-policy --log-group-name <log-group-name> --retention-in-days <retention-period>
-```
-
-Replace `<log-group-name>` with the name of the CloudWatch Log Group and `<retention-period>` with the desired retention period in days.
-
-3. Verify that the CloudWatch Log Groups are now encrypted:
-```bash
-aws logs describe-log-groups --query 'logGroups[?kmsKeyId!=null].logGroupName' --output text
-```
-
-By following these steps, you can remediate the misconfiguration of CloudWatch Log Groups not being encrypted in AWS using AWS CLI.
-
-#### Using Python
-
-To remediate the misconfiguration of CloudWatch Log Groups not being encrypted in AWS using Python, you can follow these steps:
-
-1. Install the AWS SDK for Python (Boto3) if you haven't already:
-```bash
-pip install boto3
-```
-
-2. Use the following Python script to enable encryption for all existing CloudWatch Log Groups:
-
-```python
-import boto3
-
-def encrypt_cloudwatch_logs():
-    # Initialize the CloudWatch Logs client
-    client = boto3.client('logs')
-
-    # Get a list of all existing CloudWatch Log Groups
-    log_groups = client.describe_log_groups()
-
-    # Enable encryption for each log group
-    for log_group in log_groups['logGroups']:
-        log_group_name = log_group['logGroupName']
-        
-        # Enable encryption for the log group
-        response = client.put_retention_policy(
-            logGroupName=log_group_name,
-            retentionInDays=7,
-            metricFilterName='AllLogs',
-            metricTransformations=[
-                {
-                    'metricName': 'LogEvents',
-                    'metricNamespace': 'CloudWatchLogs',
-                    'metricValue': '1'
-                }
-            ]
-        )
-
-        print(f"Enabled encryption for Log Group: {log_group_name}")
-
-if __name__ == '__main__':
-    encrypt_cloudwatch_logs()
-```
-
-3. Run the Python script, and it will enable encryption for all existing CloudWatch Log Groups in your AWS account.
-
-Please note that this script assumes you have the necessary permissions to modify CloudWatch Log Groups in your AWS account. Make sure to configure your AWS credentials properly before running the script.
-
-### Additional Reading:
\ No newline at end of file
+### Using Console:
+
+Enabling encryption from console has some limitations, you cannot associate KMS key with log group from Console.
```

Comment:
instead of saying "you cannot associate KMS key with log group from Console.", we should say "AWS does not allow KMS Key association with Log Groups from Console." and add the link next to it.

Reply:
okok